### ISSUE TITLE
Support cloning repos of all users in an org

### ISSUE THREAD

**Issue by mlaily** [gabrie30/ghorg#265]
I didn't have the chance to try this tool with a github instance, and I'm not sure how github's api behaves.

I did try it with gitlab, and when cloning an org, only the repositories contained in groups are cloned.

The personal repositories of users are not cloned, even the publicly accessible ones.

Is that intended, or is that something to fix?

**Comment by gabrie30** [comment]
Yes thats intentional if you want to clone a users repos you need to specify `--clone-type=user`

```
ghorg clone <gitlab_username> --clone-type=user --base-url=https://<your.instance.gitlab.com> --scm=gitlab --token=bGVhdmUgYSBjb21tZW50IG9uIGlzc3VlIDY2
```

This is an example from the docs [here](https://github.com/gabrie30/ghorg/blob/master/examples/gitlab.md#cloning-user-repos)

**Comment by mlaily** [comment]
Ok but this is to download the repos of a single user, right?

There is no way to clone all personal repos of all users in the org?

**Comment by gabrie30** [comment]
@mlaily are you able to install ghorg using golang? I was only able to do limited testing on a local instance so it would be great if you could test it out before I cut a new release.

**Comment by mlaily** [comment]
@gabrie30 Oh nice, I didn't expect you to implement it so quickly (or at all)!  
I just tried a local build of `master` on my instance with ~70 users and ~200 user-specific projects, and it seems to be working fine, thanks a lot!!

I didn't dig into the code to see why it's required, but having to specify `--clone-type user` as well as `ghorg clone all-users` is a bit unfortunate. I missed it on my first try and got:
```
error fetching repos for group 'all-users', error: group 'all-users' does not exist
```
which was a bit confusing. What happens if a group with this name exists?^^'

Anyway, as far as I tested it, initial clone then subsequent `fetch-all` worked fine :)  
For the record, here is the full command I tested:
```
ghorg.exe clone all-users --base-url=https://xxx.com/   --scm=gitlab --token=xxxxx   --preserve-dir --protocol=ssh --fetch-all --include-submodules --no-clean   --path "C:/git/ghorg"   --output-dir gitlab-users   --clone-type user
```
(yes, I'm on Windows)
